### PR TITLE
OS-8450 Include osirium runtime requirements

### PR DESCRIPTION
**Pull request by richlanc**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -172,3 +172,25 @@ EXPOSE 4822
 #
 CMD /usr/local/guacamole/sbin/guacd -b 0.0.0.0 -L $GUACD_LOG_LEVEL -f
```
Comment (SpeeddY) [inline Dockerfile:173]: It looks like guacd has already been started here so don't we want to remove this line if we are starting it again in `guacd.sh` ?

Comment (richlanc) [reply Dockerfile:173]: Only the last CMD actually does anything, so this is overwritten by our /guacd.sh CMD. 

I decided to leave it to keep the file as untouched as possible and we're overriding functionality at the end.

Comment (SpeeddY) [reply Dockerfile:173]: ah right ok :+1: 
"There can only be one CMD instruction in a Dockerfile. If you list more than one CMD then only the last CMD will take effect."

**Review by SpeeddY — APPROVED** [review]
lgtm

**Pull request merged**